tests/cli/log2timeline_tool.py

```diff
@@ -93,19 +95,21 @@ class Log2TimelineToolTest(test_lib.CLIToolTestCase):
                         system CPUs minus one].
 """)
 
-  def _CheckOutput(self, output, expected_output):
+  def _CheckOutput(self, output, expected_output, encoding='utf-8'):
     """Compares the output against the expected output.
 
     The actual processing time is ignored, since it can vary.
 
     Args:
       output (bytes): tool output.
-      expected_output (list[bytes]): expected tool output.
+      expected_output (list[str]): expected tool output.
```

Comment:
Consider adding a string output writer for testing.

Reply:
Ack, #1963